conf/messages

```diff
@@ -58,6 +58,7 @@ accessibility.compliance.full=This service is fully compliant with the
 accessibility.compliance.full.app=This app is fully compliant with the
 accessibility.compliance.partial=This service is partially compliant with the
 accessibility.compliance.partial.app=This app is partially compliant with the
+accessibility.compliance.openbanking=HMRC’s screens are compliant with WCAG 2.1 AA. However users visit their bank’s website or app to complete a payment. HMRC does not inspect each bank’s website or app so there might be some accessibility problems we don’t know about.
```

Comment:
Kind of feels like from a content perspective this could of been merged into the accessibility problem - and then we could have dropped the "as a result, " from the bit below and then the only change would be "it needs to be possible to have a partially compliant statement with no milestones" and hide the technical compliant bit when that's the case